lib/registration.pm

```diff
@@ -525,6 +525,9 @@ sub process_scc_register_addons {
         # start addons/modules registration, it needs longer time if select multiple or all addons/modules
         my $counter = ADDONS_COUNT;
         my @needles = qw(import-untrusted-gpg-key nvidia-validation-failed yast_scc-pkgtoinstall yast-scc-emptypkg inst-addon contacting-registration-server refreshing-repository system-probing);
+
+        # In SLE Micro, we don't need to continue with registering any additional module.
+        return if is_sle_micro;
```

Comment:
We might still want to register livepatching or not?

Reply:
I had to add this `return` because the installation was not asking for it and the test got stuck waiting for license needle...